code/datums/status_effects/debuffs.dm

```diff
@@ -289,6 +294,10 @@
 /datum/status_effect/cling_tentacle/on_remove()
 	REMOVE_TRAIT(owner, TRAIT_IMMOBILIZED, "[id]")
 
+/datum/status_effect/cling_tentacle/batterer
+	id = "cling_tentacle_batterer"
+	alert_type = null
+	duration = 7 SECONDS
```

Comment:
sorry I didnt review this before now. wayyy too late no requirement to change these at all but, why not call `Immobilize()`

Reply:
no real reason, just didn't think of it at the time.
i'm gonna be touching the batterer later this week as it's actually in so i'll change this